.github/workflows/release.yaml

```
@@ -116,3 +116,17 @@ jobs:
         files: out/*
         body_path: ${{ env.RELEASE_TAG }}.md
         tag_name: ${{ env.RELEASE_TAG }}
+
+  build_ipam:
+    needs: push_release_tags
+    name: Build IPAM container image
+    if: github.repository == 'metal3-io/ip-address-manager'
+    uses: metal3-io/project-infra/.github/workflows/container-image-build.yml@main
+    with:
+      image-name: "ip-address-manager"
+      pushImage: true
+      ref: ${{ needs.push_release_tags.outputs.release_tag }}
```

```suggestion
      ref: ${{ env.RELEASE_TAG }}
```
We can also use env.RELEASE_TAG here right?

---

Not really. The `release` job isn't configured to run before `build_ipam`. They could be running in parallel, or in any order. Only the `push_release_tag` is guaranteed to run first. Also, I'm not sure if the env is maintained between the jobs.

Is there any reason for using env instead of outputs?